src/assets/locales/fr.json

```diff
@@ -347,36 +381,43 @@
   "preferences-screen.material-quality-setting.medium": "Medium",
   "preferences-screen.preference.allow-multiple-hubs-instances": "Désactiver la sortie automatique quand de multiples instances de hubs sont ouvertes",
-  "preferences-screen.preference.animate-waypoint-transitions": "Animer les transitions entre les waypoint",
+  "preferences-screen.preference.animate-waypoint-transitions": "Animer les transitions entre les waypoints",
+  "preferences-screen.preference.audio-clipping-threshold": "Seuil pour l'audio clipping",
   "preferences-screen.preference.disable-auto-gain-control": "Désactiver le contrôle automatique du gain du micro",
   "preferences-screen.preference.disable-auto-pixel-ratio": "Désactiver les ajustements automatiques du ratio de pixels",
   "preferences-screen.preference.disable-backwards-movement": "Désactiver les mouvements en marche arrière",
   "preferences-screen.preference.disable-echo-cancellation": "Désactiver l'annulation d'écho du micro",
   "preferences-screen.preference.disable-idle-detection": "Désactiver la sortie automatique si inactif ou en arrière-plan",
   "preferences-screen.preference.disable-movement": "Désactiver le mouvement",
   "preferences-screen.preference.disable-noise-suppression": "Désactiver la suppression de bruit du micro",
+  "preferences-screen.preference.disable-panning": "Désactiver le panning audio gauche/droite",
   "preferences-screen.preference.disable-sound-effects": "Désactiver les effets sonores",
   "preferences-screen.preference.disable-strafing": "Désactiver le mitraillage",
   "preferences-screen.preference.disable-teleporter": "Désactiver la téléportation",
+  "preferences-screen.preference.enable-audio-clipping": "Activer l'audio clipping",
   "preferences-screen.preference.enable-dynamic-shadows": "Activer les ombres dynamiques",
   "preferences-screen.preference.enable-gyro": "Activer le gyroscope (si compatible avec le navigateur / l'appareil)",
   "preferences-screen.preference.enable-on-screen-joystick-left": "Activer le joystick sur l'écran à gauche pour se déplacer",
   "preferences-screen.preference.enable-on-screen-joystick-right": "Activer le joystick sur l'écran à droite pour se déplacer",
+  "preferences-screen.preference.fast-room-switching": "Activer le basculement rapide de salle",
   "preferences-screen.preference.global-media-volume": "Volume audio des médias",
   "preferences-screen.preference.global-voice-volume": "Volume audio de la voix entrante",
   "preferences-screen.preference.invert-touchscreen-camera-move": "Inverser la direction du mouvement de caméra pour les écrans tactiles",
+  "preferences-screen.preference.lazy-load-scene-media": "Activer le chargement différé des ressources média de la scène",
   "preferences-screen.preference.locale": "Langue",
-  "preferences-screen.preference.material-quality-setting": "Qualité du matériau",
+  "preferences-screen.preference.material-quality-setting": "Qualité des surfaces (materials)",
```

Comment:
"Qualité des surfaces (textures)" ?

Reply:
In English it's "Material quality". In the threejs sense, it's changing MeshStandardMaterial to MeshBasicMaterial. Material uses textures.
In French we can probably just use "Qualité des textures"?

Reply:
I replaced the translation by "Qualité des textures" in https://github.com/mozilla/hubs/pull/5002/commits/8d5800cd789a646f732c1d676c353752da736b13